e2e/jest.test-setup.ts

```diff
@@ -128,45 +152,69 @@ beforeEach(async () => {
   });
 
   page.on('requestfinished', async (request) => {
-    if (canLogResponse(request)) {
-      if (isApiFailure(request)) {
-        await logError(request);
-      } else {
-        await logResponse(request);
+    let method;
+    let url;
+    let status;
+    try {
+      if (canLogResponse(request)) {
+        // Try find out what the request was if exception thrown.
+        method = request.method();
+        const resp = request.response();
+        url = resp.url();
+        status = resp.status();
+
+        if (isApiFailure(request)) {
+          await logError(request);
+        } else {
+          const responseBody = await transformResponseBody(request);
+          if (skipApiResponseBody(request)) {
+            console.debug('❗ Request finished: ' +
+               `${request.response().status()} ${request.method()} ${request.url()}`);
```

Comment:
nit: you can replace these method calls with the new local vars, e.g.

`${status} ${method} ${url}`

(Also applies to the console.debug line below

Reply:
👍